fabfile/run.py

```diff
@@ -13,7 +13,7 @@
 import ConfigParser
 import time
 
-__all__ = ['jdbc', 'regression', 'pgbench_tests', 'tpch_automate', 'valgrind', 'valgrind_filter_put_results']
+__all__ = ['jdbc', 'regression', 'pgbench_tests', 'tpch_automate', 'valgrind_multi', 'valgrind_multi_1', 'valgrind_columnar']
```

Comment:
could instead make use of the `run.valgrind:<option>` syntax to pass the test target